### PR TITLE
feat(FWS-78):Ability To Create An Invoice

### PR DESCRIPTION
**Pull request by AlexeiTimbro**
**JIRA: link to jira ticket**
https://champlainsaintlambert.atlassian.net/browse/FWS-62

**What is the ticket about and why are we making this change?**

This ticket concerns being able to create an invoice as a trainer/admin on a client. This is important because as a trainer you want to be able to add an invoice to a client that hasn't paid for the fitness appointment that they had with the trainer. This is a way of reminding a client that they haven't paid yet.

**What are the various changes and what other modules do those changes affect?**
 
-2 types of testing
-integrated testing will be done in my next PR due to a deployment issue
-Selenium Test
-backend for the addInvoice
-Front end for the addInvoice
-Added both Admin/Trainer CreateInvoice form to be able to create the invoice
-Added both the create button that brings you to the create form.

**UI Change:**

Before Invoice view:
image

After Invoice View:
image

After clicking Create Invoice:
image
image

After creating the invoice ( invoice added):
image

### CODE REVIEW

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateInvoice.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+                {/* Price */}
+                <label htmlFor="price">Price:</label>
+                <input type="text" id="price" name="price" value={newInvoice.price} onChange={handleInputChange} />
+
+                <button type="submit">Create Invoice</button>
+            </form>
+        </div>
+    );
+}
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateInvoice.js:153]: You forgot localization

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminInvoices.js [hunk 2]
```diff
@@ -108,4 +121,4 @@ function AdminInvoices() {
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/AdminPanelPage/AdminInvoices.js:121]: You forgot localization

File: fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerCreateInvoice.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+                {/* Price */}
+                <label htmlFor="price">Price:</label>
+                <input type="text" id="price" name="price" value={newInvoice.price} onChange={handleInputChange} />
+
+                <button type="submit">Create Invoice</button>
+            </form>
+        </div>
+    );
+}
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerCreateInvoice.js:153]: You forgot localization

File: fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerInvoices.js [hunk 4]
```diff
@@ -104,6 +118,7 @@ function TrainerInvoices() {
         </div>
 
 
+
     );}
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerInvoices.js:123]: You forgot localization

**Review by Eli293 — APPROVED** [review]
Looks good.

**Review by Abdur-Siddiqui — COMMENTED** [review]
looks good

**Review by Abdur-Siddiqui — APPROVED** [review]
looks good

**Pull request merged**